CHANGELOG.md

```diff
@@ -7,6 +7,70 @@ documentation before upgrading to a new release.
+troubleshooting of issues in this area. It also addresses several crashes,
+in the core itself and both in Icinga DB and IDO (numbers out of range).
+In addition, it fixes several other issues such as lost notifications
+or TimePeriod/ScheduledDowntime exceeding specified date ranges.
+
+### Crash Fixes
+
+* Invalid `DateTime#format()` arguments in config and console on Windows Server 2016 and older. #10112
+* Downtime scheduling at runtime with non-existent trigger. #10049
+* Object creation at runtime during Icinga DB initialization. #10151
+* Comment on a service of a non-existent host. #9861
+
+### Miscellaneous Bugfixes
+
+* Lost notifications after recovery outside the notification time period. #10187
+* TimePeriod/ScheduledDowntime exceeding specified date range. #9983 #10107
+* Clean up failure for obsolete Downtimes. #10062
+* ifw-api check command: use correct process-finished handler. #10140
+* Email notification scripts: strip 0x0D (CR) for a proper Content-Type. #10061
+* Several fixes and improvements of the code quality. #10066 #10214 #10254 #10263 #10264
+
+### Cluster and API
+
+* Sync runtime objects in topological order to honor their dependencies. #10000
+* Make parallel config syncs more robust. #10013
+* After object creation via API fails, clean up properly for the next try. #10111
+* Close HTTPS connections properly to prevent leaks. #10005 #10006
+* Reduce the number of cluster messages in memory at the same time. #9991 #9999 #10210
+* Once a cluster connection shall be closed, stop communicating. #10213 #10221
+* Remove unnecessary blocking of semaphores. #9992 #9994
+* Reduce unnecessary cluster messages setting the next check time. #10011
+
+### Icinga DB and IDO
+
+* IDO: fix object relations after aborted synchronization. #10065
+* Icinga DB, IDO: limit all timestamps to four year digits. #10058 #10059
+* Icinga DB: limit execution\_time and latency (milliseconds) to database schema. #10060
+
+### Troubleshooting
+
+* Add `/v1/debug/malloc_info` which calls `malloc_info(3)` if available. #10015
+* Add log messages about own network I/O. #9993 #10141 #10207
+* Several fixes and improvements of log messages. #9997 #10021 #10209
+
+### Windows
+
+* Update OpenSSL shipped on Windows to v3.0.15. #10170
+* Update Boost shipped on Windows to v1.86. #10114
+* Support CMake v3.29. #10037
+* Don't require to build .msi as admin. #10137
+* Build configuration scripts: allow custom `$CMAKE_ARGS`. #10312
+
+### Documentation
+
+* Distributed Monitoring: add section "External CA/PKI". #9825
+* Explain how to enable/disable debug logging on the fly. #9981
+* Update supported OS versions and repository configuration. #10064 #10090 #10120 #10135 #10136
+* Several fixes and improvements. #9960 #10050 #10071 #10156 #10194
+* Replace broken links. #10115 #10118 #10282
+* Fix typographical and similarly trivial errors. #9953 #9967 #10056 #10116 #10152 #10153 #10204
```

Comment:
Especially this list (but not only) would look very monotonic then. What about giving the support branch PRs as before, but also the original ones in ( ), e.g:

```suggestion
* Distributed Monitoring: add section "External CA/PKI". #10081 (#9825)
* Explain how to enable/disable debug logging on the fly. #10081 (#9981)
* Update supported OS versions and repository configuration. #10081 (#10064 #10090 #10120 #10135 #10136)
* Several fixes and improvements. #9960 #10050 #10071 #10156 #10081 (#10194)
* Replace broken links. #10081 (#10115 #10118 #10282)
* Fix typographical and similarly trivial errors. #10081 (#9953 #9967 #10056 #10116 #10152 #10153 #10204)
```

Reply:
Maybe another reason to avoid such monster backport PRs like #10081. That must have been a pain to review.

In this case, I'd just give the reference once before the list.